src/modules/createpopup.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable */
+import { renderComment,generateComment } from "./rendercomment.js";
+import commentsCounter from "./commentscounter.js";
+
+export const createPopUpDetails = (details) => {
+  /* eslint-enable */
```

Comment:
- [ ] I notice that you have disabled your linters in multiple places of your codebase. Please avoid doing this. Linters are important because they ensure a consistent code format and enforce some of the best practices, providing, as result, good quality code. Please instead of disabling the linter, correct the rules! 🤓 🛠️

Reply:
[Problem solved](https://github.com/Wahaj-Ali/tv-hub/commit/f35c250b19dda085593737e0e0ad36b3404305bc)